tests/visualisations/test_trackmap.py

```diff
@@ -19,16 +28,109 @@ def prefilled_trackmap(example_plottingdataframe):
 
 
 def test_TrackMap(emptytrackmap, prefilled_trackmap, example_plottingdataframe):
-    objects_in_empty = len(emptytrackmap.__dict__['_children'])
-    objects_in_nonempty = len(prefilled_trackmap.__dict__['_children'])
+    objects_in_empty = len(emptytrackmap._children)
+    objects_in_nonempty = len(prefilled_trackmap._children)
     assert objects_in_empty == 1, 'Should contain only the aerial photo'
-    assert objects_in_nonempty == 1 + len(example_plottingdataframe), 'Should contain the aerial photo and some markers'
+    assert objects_in_nonempty == 1 + len(
```

Comment:
Bit of a misleading test based on the number of children

Reply:
Agreed, this is a fixed 2 for the linestrings, with 1 added for the aerial photograph. This is now made explicit.